src/test/java/com/nvidia/spark/rapids/jni/CastStringsTest.java

```diff
@@ -193,18 +193,71 @@ void castToDecimalNoStripTest() {
     }
   }
 
-  private void convTestInternal(Table input, Table expected, int fromBase) {
+  private void baseConversionHelper(Table input, Table expected, int fromBase,
+    boolean dropLeadingZeros) {
```

Comment:
Should we consider a more descriptive name for this helper?  This seems to be testing the round-trip from base-10 to both base-10 and base-16.

Perhaps `testRoundTripToBase10And16`?